Models/StandUpReminder.js

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose'
```

Comment:
Please, please, please, move the standupreminder to `StandUp` model! Thanks @huilunang :)

Reply:
More like 'reminders'

- If users need to have personal reminders then, that can be stored on the user model! 
- The default reminders for the standup should also be stored on the standup model
- This means, that you would have to modify the reminders job to send our reminders based on user subscriptions.